src/modules/auth/index.js

```diff
@@ -170,6 +173,27 @@ export default class AuthModule extends Module {
 		});
 	}
 
+	logoutOnExpiredSession (that, al=false) {
```

Comment:
It is not the logout after expired session - it is a warning that that the session is going to expire, or is expired.